main.rb

```diff
@@ -0,0 +1,48 @@
+morse_code = {
+  '.-' => 'A',
+  '-...' => 'B',
+  '-.-.' => 'C',
+  '-..' => 'D',
+  '.' => 'E',
+  '..-.' => 'F',
+  '--.' => 'G',
+  '....' => 'H',
+  '..' => 'I',
+  '.---' => 'J',
+  '-.-' => 'K',
+  '.-..' => 'L',
+  '--' => 'M',
+  '-.' => 'N',
+  '---' => 'O',
+  '.--.' => 'P',
+  '--.-' => 'Q',
+  '.-.' => 'R',
+  '...' => 'S',
+  '-' => 'T',
+  '..-' => 'U',
+  '...-' => 'V',
+  '.--' => 'W',
+  '-..-' => 'X',
+  '-.--' => 'Y',
+  '--..' => 'Z'
+}
+
+def decode_char(string, morse_code)
+  puts morse_code[string]
+end
+
+def decode_word(morse, morse_code)
+  decoded_chars = morse.split.map { |char| morse_code[char] }
+  decoded_word = decoded_chars.join
+  puts decoded_word
+end
```

Comment:
- [ ] I would highly recommend returning the methods output instead of printing it, this will make it easy to use your code in different scenarios. Also, this will make it possible to return the string without the use of `print` in the `decoded_message` method

```suggestion
def decode_char(string, morse_code)
   morse_code[string]
end

def decode_word(morse, morse_code)
  decoded_chars = morse.split.map { |char| morse_code[char] }
  decoded_word = decoded_chars.join
   decoded_word
end
```